practice-two/src/themes/toasts.js

```diff
@@ -0,0 +1,4 @@
+export const toastTheme = {
+  success: "#5cb85c",
```

Comment:
defind and reuse the color variable

Reply:
I changed it to variable css
![image](https://github.com/datdragon777/React-Training/assets/74806826/f91b9e82-4731-4063-a162-07b56a4828b5)
![image](https://github.com/datdragon777/React-Training/assets/74806826/39de07de-1b76-49b9-bfd5-17fc9d35f88a)